packages/bpk-docs/src/layouts/DocsLayout/DocsLayout.js

```diff
@@ -20,192 +20,22 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 import SideNavLayout from './../SideNavLayout';
-import NeoSideNav from './../NeoSideNav';
-import * as routes from './../../constants/routes';
+import NeoSideNavLayout from './../NeoSideNavLayout';
 
-const links = [
```

Comment:
We introduce different behaviour for the side nav in tablet vs desktop breakpoints. In tablet, choosing a new section doesn't navigate. In desktop it does.

To enable this, we list the knowledge of links to a separate place so that each section knows how to render each others links.